### PR TITLE
feat: add cymbalshop e2e test

### PR DESCRIPTION
**Pull request by caetano-colin**

### CODE REVIEW

File: build/int.cloudbuild.yaml [hunk 1]
```diff
@@ -162,12 +162,20 @@ steps:
       - appinfra-apply
       - fleetscope-verify
 
-  - id: app-e2e
+  - id: cymbal-bank-e2e
     name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
     args:
       ["/bin/bash", "-c", "cft test run TestAppE2E --stage verify --verbose"]
```
Comment (apeabody) [inline build/int.cloudbuild.yaml:168]: It would be nice to rename the test from `TestAppE2E` > `TestCymbalBankE2E` for clarity/consistency

**Review by apeabody — APPROVED** [review]
Thanks @caetano-colin!

One suggestion, otherwise this LGTM

**Review by gtsorbo — APPROVED** [review]

**Comment by apeabody** [comment]
```
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:   "membershipStates": {
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:     "projects/31855676613/locations/us-central1/memberships/cluster-us-central1-development": {
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:       "servicemesh": {
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:         "controlPlaneManagement": {},
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:         "dataPlaneManagement": {}
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:       },
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:       "state": {
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:         "code": "ERROR",
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:         "description": "Unable to connect to cluster.",
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:         "updateTime": "2024-11-21T19:01:58.220285610Z"
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:       }
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:     }
Step #9 - "fleetscope-verify": TestFleetscope/development 2024-11-21T19:07:06Z command.go:185:   },
```

**Pull request merged**